frontend/src/components/abfragevarianten/BedarfsmeldungFachreferateDialog.vue

```diff
@@ -157,12 +157,20 @@ export default class BauvorhabenDataTransferDialog extends Mixins(
   private uebernehmenBedarfsmeldung(): void {
     const validationMessage: string | null = this.findFaultInBedarfsmeldung(this.bedarfsmeldung);
     if (_.isNil(validationMessage)) {
```

Comment:
Von der Reihenfolge her sollten zuerst die Client Validatoren vor der fachl. Prüfung ausgeführt werden, d. h. _validateDialogForm_ sollte vor _.isNil(validationMessage) geprüft werden.

Beispiel: views/Abfrage.vue

![grafik](https://github.com/it-at-m/isi-frontend/assets/119875041/9c8054bf-2b53-4928-8174-a1f6f19b7dbd)

Reply:
Hab ich zuerst so gemacht, dann kommen aber nicht die gewünschten Fehlermeldungen. Was ist wichtiger?